preset.ts

```diff
@@ -197,33 +200,50 @@ class SvelteJestAdder extends Adder {
     Preset
       .editJson('jest.config.json').merge({
         transform: {
-          '^.+\\.svelte$': ['svelte-jester', {preprocess: true}],
+          '^.+\\.svelte$': ['./node_modules/svelte-jester/dist/transformer.mjs', {preprocess: true}],
```

Comment:
Can this not be just `svelte-jester`?

Reply:
Ah, I just saw your explanation on https://github.com/rossyman/svelte-add-jest/issues/19#issuecomment-916218761

Reply:
Yeah it's a rather peculiar issue. I would've assumed that Jest would've somehow picked up the `.mjs` file automagically when the ESM flag is enabled, but it seems to be forcing the `.cjs` transformer usage.